vm/vmcore/guestmem/src/ranges.rs

```diff
@@ -296,6 +296,18 @@ impl<'a> MemoryWrite for PagedRangeWriter<'a> {
         Ok(())
     }
 
+    fn fill(&mut self, val: u8, len: usize) -> Result<(), AccessError> {
+        let range = self
+            .range
+            .try_subrange(0, len)
+            .ok_or_else(|| AccessError::OutOfRange(self.len(), len))?;
+        self.mem
+            .fill_range(&range, val)
+            .map_err(AccessError::Memory)?;
+        self.range.skip(len);
+        Ok(())
+    }
+
     fn zero(&mut self, len: usize) -> Result<(), AccessError> {
```

Comment:
Remove special case

Reply:
Ah, I figured that it might be worthwhile to use `GuestMemory`'s two distinct APIs for zero vs. fill, in case the zero case could be optimized.

...but then again, if it could just be optimized, it'd simply be done by specializing the fill implementation based on if the number of zero.

so yeah, will remove